src/mlpack/core/data/scaler_methods/pcawhitening.hpp

```diff
@@ -0,0 +1,136 @@
+/**
+ * @file pcawhitening.hpp
```

Comment:
Tiny issue, just to match the rest of the codebase I'd suggest `pca_whitening.hpp`. :+1:

Reply:
Done.